soul-plugin/soul-plugin-sign/src/test/java/org/dromara/soul/plugin/sign/service/DefaultSignServiceTest.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dromara.soul.plugin.sign.service;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import org.apache.commons.lang3.tuple.Pair;
+import org.dromara.soul.common.constant.Constants;
+import org.dromara.soul.common.dto.AppAuthData;
+import org.dromara.soul.common.dto.AuthPathData;
+import org.dromara.soul.common.dto.PluginData;
+import org.dromara.soul.common.enums.PluginEnum;
+import org.dromara.soul.common.utils.SignUtils;
+import org.dromara.soul.plugin.api.SignService;
+import org.dromara.soul.plugin.api.context.SoulContext;
+import org.dromara.soul.plugin.api.result.SoulResultEnum;
+import org.dromara.soul.plugin.base.cache.BaseDataCache;
+import org.dromara.soul.plugin.sign.cache.SignAuthDataCache;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mock.http.server.reactive.MockServerHttpRequest;
+import org.springframework.mock.web.server.MockServerWebExchange;
+import org.springframework.web.server.ServerWebExchange;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * DefaultSignService Test.
+ *
+ * @author Phoenix Luo
+ * @version 2020/12/5
+ **/
+@RunWith(MockitoJUnitRunner.class)
+public class DefaultSignServiceTest {
```

Comment:
add final on the class

Reply:
OK, Thanks ,I checked code style in the local, but don't tell me test class must be `final`. I'll pay attention to that later.
In locally, I did not got many error by use ’ mvn clean install -Dmaven.javadoc.skip=true‘. 
I'll test it again

Reply:
yes, this is a appointment but not int the code style